build/Helix/RunTestsInHelix.proj

```diff
@@ -4,8 +4,14 @@
     <HelixType>test/product/</HelixType>
     <EnableXUnitReporter>true</EnableXUnitReporter>
     <EnableAzurePipelinesReporter>true</EnableAzurePipelinesReporter>
-    <FailOnMissionControlTestFailure>true</FailOnMissionControlTestFailure>
     <HelixPreCommands>$(HelixPreCommands);set testnameprefix=$(Configuration).$(Platform);set testbuildplatform=$(Platform)</HelixPreCommands>
+    
+    <!-- We'll output a custom error message that explicitly lists the failing tests,
+         so we don't want Helix to output a standard error message, as it's not nearly
+         as helpful - it just says "an error occurred", essentially. -->
+    <FailOnMissionControlTestFailure>false</FailOnMissionControlTestFailure>
+    <FailOnTestFailure>false</FailOnTestFailure>
+    <FailOnWorkItemFailure>false</FailOnWorkItemFailure>
```

Comment:
I'm worried about these. If there is some error from Helix that does not surface in the form of test result failures, we could end up silently ignoring them. E.g. if the HelixWorkItems failed to get scheduled or something.

Reply:
That's fair - this is just a nice-to-have thing that prevented us from getting unhelpful noise in our error logs, so I can undo this change if you think there's a practical reason not to do this.